recipe/build.sh

```diff
@@ -1,2 +1 @@
-#!/bin/bash
```

Comment:
What version of the [helper script](https://github.com/bgruening/conda_r_skeleton_helper) did you use? I don't think the shebang line should be removed from the build script.